src/Microsoft.DotNet.CodeFormatting/Rules/UsingLocationRule.cs

```diff
@@ -12,7 +12,7 @@ namespace Microsoft.DotNet.CodeFormatting.Rules
     /// <summary>
     /// This will ensure that using directives are placed outside of the namespace.
     /// </summary>
-    [SyntaxRule(Name = UsingLocationRule.Name, Description = UsingLocationRule.Description, Order = SyntaxRuleOrder.UsingLocationFormattingRule)]
+    [SyntaxRule(Name = UsingLocationRule.Name, Description = UsingLocationRule.Description, Order = SyntaxRuleOrder.UsingLocationFormattingRule, DefaultRule = false)]
```

Comment:
When I was cherry-picking the four commits in this PR from another local branch onto my local anfix branch, this line came up as a conflict, and I chose the version with DefaultRule=false. It looks like that was wrong and that I had no reason to change this file.